nav2_map_server/include/nav2_map_server/map_mode.hpp

```diff
@@ -1,16 +1,18 @@
-// Copyright 2019 Rover Robotics
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
+/**
```

Comment:
I think the copyright check doesn't like copyright headers with block comments; it only wants // style line comments. 

I'd just leave the comment block as is. I don't see any real need to include the copyright header in the doxygen output

Reply:
Cool! I will revert them back

Reply:
@crdelsey can you re-review